### ISSUE TITLE
I'm unable to setup the pathMapping / Sourcemaps Overrides option

### ISSUE THREAD

**Issue by SergiArias** [microsoft/vscode-edge-devtools#228]
As I understand from the documentation, to setup the mappings I sould use:

```json
    "vscode-edge-devtools.pathMapping": {
      "/": "/Users/sergiarias/ActiveCaramel/website/"
    },
```
first parameter being the root url (in my case localhost:3000) and second the folder where it points in my computer

But I'm not having any luck in opening the scss files with this setup. 

Going to the browser and revealing sources I get this link though: https://localhost:3000/Users/sergiarias/ActiveCaramel/artcants2020/wp-content/themes/sage-artcants2020/resources/assets/styles/resources/assets/styles/caramel/_bootstrap-cc.scss

So I tried 
```json
    "vscode-edge-devtools.pathMapping": {
      "/": ""
    },
```

and 
```json
    "vscode-edge-devtools.pathMapping": {
      "/": "/"
    },
```

Anyway, I think I am getting confused with the sourcemaps overrides option. But I don't know how to correctly map the webpack sourcemaps with the result file.

Can somebody help me with this mapping based on the source link? Or do I have to take a look how webpack options are defined?

**Comment by SergiArias** [comment]
I think I am getting confused with the Sourcemaps option...

**Comment by vidorteg** [comment]
Hi @SergiArias 

I think this is a duplicate of this bug: #184 
The behavior was fixed in here: #221 but we have not released a version that contains the fix (will do soon)

In the meanwhile you can workaround this issue by using absolute paths:
```
"vscode-edge-devtools.pathMapping": {
        "/": "C:\\testproject"
    }
```
